rhizome/host/bin/prep_host.rb

```diff
@@ -92,6 +92,9 @@
 # We need smartmontools, nvme-cli and jq in order to probe the disk status of VmHosts
 r "apt-get -y install smartmontools nvme-cli jq"
 
+# aria2 is able to download a file from signed URLs concurrently
+r "apt-get -y install aria2"
```

Comment:
```suggestion
r "apt-get -y install aria2" if use_aria2
```

Unless there is a reason to install it if it won't be used?

Reply:
I updated this PR to use `htcat` instead of `aria2`